src/xmlattribute.h

```diff
@@ -93,13 +93,13 @@ class XmlAttribute
     bool isType(const char *attributeName);
```

Comment:
Might want to replace `attributeName` with just `name`.

Same in the other signature, in addition to replacing `attributeNamespace` with `namespace`.